### PR TITLE
Add a tuning example to the README

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +44,39 @@ The feature selector can be re-instantiated with a longer time limit.
 >>> X_new = SelectFromQuadraticModel(num_features=20, time_limit=200).fit_transform(X, y)
 ```
 
+### Tuning
+
+It is possible to use `SelectFromQuadraticModel` with scikit-learn's
```
Comment (JoelPasvolsky) [inline README.md:49]:
```suggestion
You can use `SelectFromQuadraticModel` with scikit-learn's
```

Ridiculously minor, feel free to ignore

File: README.md [hunk 2]
```diff
@@ -42,6 +44,39 @@ The feature selector can be re-instantiated with a longer time limit.
...
+[hyper-parameter optimizers](https://scikit-learn.org/stable/modules/classes.html#hyper-parameter-optimizers).
+
+For example, the number of features can be tuned using a grid search. **Please note that this will
+submit many problems to the hybrid solver.**
```
Comment (JoelPasvolsky) [inline README.md:53]: Consider whether it's worth making this more specific: about twenty problems of 5 seconds each for a total of 100 seconds of hybrid usage time?

**Review by JoelPasvolsky — APPROVED** [review]
LGTM!

File: README.md (hunk 2 above)
Comment (arcondello) [reply README.md:53]: Unfortunately it's not deterministic I don't think.

**Pull request merged**